### PR TITLE
minor: skip ArchUnit Testing until #13544

### PR DESCRIPTION
**Pull request by Kevin222004**
minor: skip ArchUnit Testing

### CODE REVIEW

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Please do:

1. Create issue to re-enable tests
2. Disable tests by `Disabled` annotation, with value of "until..." instead of disablement by pom.xml

File: config/archunit-store/slices-should-be-free-of-cycles-suppressions [hunk 1]
```diff
@@ -11,8 +11,6 @@ Cycle detected: Slice api -> \
     - Method <com.puppycrawl.tools.checkstyle.api.AbstractCheck.log(int, int, java.lang.String, [Ljava.lang.Object;)> calls method <com.puppycrawl.tools.checkstyle.utils.CommonUtil.lengthExpandedTabs(java.lang.String, int, int)> in (AbstractCheck.java:245)\
     - Method <com.puppycrawl.tools.checkstyle.api.AbstractCheck.log(com.puppycrawl.tools.checkstyle.api.DetailAST, java.lang.String, [Ljava.lang.Object;)> calls method <com.puppycrawl.tools.checkstyle.utils.CommonUtil.lengthExpandedTabs(java.lang.String, int, int)> in (AbstractCheck.java:275)\
   2. Dependencies of Slice utils\
-    - Method <com.puppycrawl.tools.checkstyle.utils.AnnotationUtil.containsAnnotation(com.puppycrawl.tools.checkstyle.api.DetailAST)> has parameter of type <com.puppycrawl.tools.checkstyle.api.DetailAST> in (AnnotationUtil.java:0)\
```
[the remark this answers is not in the document]
Comment (Kevin222004) [reply config/archunit-store/slices-should-be-free-of-cycles-suppressions:14]: I have created it for testing.

I will update in next push

**Comment by romani** [comment]
Create issue to enable, assign it to project and your self to not forget.

Put link to issue in commit message like `minor: .... Untill #xxxx`

File: config/archunit-store/slices-should-be-free-of-cycles-suppressions (hunk 1 above)
[the remark this answers is not in the document]
Comment (Kevin222004) [reply config/archunit-store/slices-should-be-free-of-cycles-suppressions]: done

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/ArchUnitCyclesCheckTest.java [hunk 2]
```diff
@@ -29,6 +30,7 @@
 import com.tngtech.archunit.lang.ArchRule;
 import com.tngtech.archunit.library.freeze.FreezingArchRule;
 
+@Disabled("Skip until Summer2023")
```
Comment (nrmancuso) [inline src/test/java/com/puppycrawl/tools/checkstyle/internal/ArchUnitCyclesCheckTest.java:33]: `@Disabled("until https://github.com/checkstyle/checkstyle/issues/13544")

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

**Comment by Kevin222004** [comment]
ci not related to changes

**Review by nrmancuso — APPROVED** [review]
Ok to merge as CI passes

**Pull request merged**